packages/synapse-react-client/src/mocks/team/mockTeam.ts

```diff
@@ -157,4 +181,28 @@ export const mockTeamMembershipStatuses: readonly Readonly<TeamMembershipStatus>
       hasUnmetAccessRequirement: false,
       canSendEmail: false,
     },
+    {
+      teamId: mockTeamData5.id,
+      userId: String(MOCK_USER_ID),
+      isMember: false,
+      hasOpenInvitation: false,
+      hasOpenRequest: true,
+      canJoin: false,
+      membershipApprovalRequired: true,
+      hasUnmetAccessRequirement: false,
+      canSendEmail: false,
+    },
+  ]
+
+export const mockTeamMembershipInvitations: readonly Readonly<MembershipInvitation>[] =
+  [
+    {
+      id: uniqueId(),
+      createdOn: new Date().toISOString(),
+      createdBy: String(MOCK_USER_ID_2),
+      teamId: mockTeamData4.id,
+      inviteeId: String(MOCK_USER_ID),
+      message:
+        'Come join my cool team so we can submit to the challenge together!',
+    },
```

Comment:
Team 4 has an open invitation to the mock user, so we can later validate the flow in PORTALS-2940